api/src/main/java/org/pmiops/workbench/workspaces/WorkspaceServiceImpl.java

```diff
@@ -174,7 +178,7 @@ public WorkspaceResponse getWorkspace(String workspaceNamespace, String workspac
     DbWorkspace dbWorkspace = workspaceDao.getRequired(workspaceNamespace, workspaceId);
     FirecloudWorkspaceResponse fcResponse;
     FirecloudWorkspaceDetails fcWorkspace;
-
+    validateControlledTierAccess(dbWorkspace);
```

Comment:
We can make this work for whatever tier the workspace is in.  Please also move this up to 179